ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -74,11 +74,13 @@ conformance_test(
     test_tool_args = [
         "--verbose",
         "--concurrent-test-runs=4",  # lowered from default #procs to reduce flakes - details in https://github.com/digital-asset/daml/issues/7316
-        # The following three contract key tests require uniqueness and are tested in conformance-test-contract-keys below
+        # The following three contract key tests require uniqueness
         "--exclude=ContractKeysIT,ContractKeysIT:CKFetchOrLookup,ContractKeysIT:CKNoFetchUndisclosed,ContractKeysIT:CKMaintainerScoped" +
         ",ParticipantPruningIT" +  # see "conformance-test-participant-pruning" below
         ",ConfigManagementServiceIT,LedgerConfigurationServiceIT" +  # dynamic config management not supported by Canton
-        ",ClosedWorldIT",  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
+        ",ClosedWorldIT" +  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
+        # Some of gRPC status codes changed from INVALID_ARGUMENT to ABORTED (see https://github.com/digital-asset/daml/pull/9218):
+        ",SemanticTests:SemanticDoubleSpendBasic,SemanticTests:SemanticDoubleSpendShared,SemanticTests:SemanticPrivacyProjections,SemanticTests:SemanticDivulgence",
```

Comment:
Where do we track the fixing of the excludes added here?

Reply:
No idea, my team is not responsible for either canton or the ledger api test tool :shrug:

Reply:
Maybe @oliverse-da tracks this somewhere

Reply:
Yes, I track these in canton. Let me take a look at these failing tests.

Reply:
These tests are likely failing on canton because the canton build has embedded inside it an older version of the ledger api server, i.e. with the next canton release and upgrade in the daml repo, these test should pass again given that it is the ledger api server's `ApiSubmissionService` that now returns `ABORTED`.

Reply:
Also, created `KVL-876` as a follow-up.

Reply:
Thanks @hubert-da

Reply:
Follow-up happening here: https://github.com/digital-asset/daml/pull/9356